src/data_structure/temporal_structure.jl

```diff
@@ -344,6 +345,28 @@ function to_time_slice(m::Model; t::TimeSlice)
     unique(Iterators.flatten((in_blocks, in_gaps)))
 end
 
+"""
+    rep_time_slice_mapping(m::Model)
+Generate an `Array` mapping all non-representative to representative time-slices
+"""
+function rep_time_slice_mapping(m::Model)
+    rep_dict=Dict()
+    for blk in indices(ordering_periods)
+        for t_start_real in ordering_periods(temporal_block=blk).indexes
```

Comment:
We'd need a check that `ordering_periods` is a TimeSeries to be able to use `indexes`. But also I'm not a fan of accessing fields directly...